proposals/NNNN-swiftpm-extensible-build-tools.md

```diff
@@ -304,6 +307,8 @@ protocol CommandConstructor {
     /// Note that input and output dependencies are ignored for prebuild and
     /// postbuild actions, since they always run before and after the build
     /// respectively.
+    
+    // [Tom] is "addCommand" or "registerCommand" more correct?
```

Comment:
IMO the language here is somewhat nuanced.  This is a command that's being generated/created here, and will be run later.  The "register" for the derived source file is because the file isn't actually being created or added here — it will be created by the tool, and here SwiftPM is just being told about it.  But it might make sense to have "add" here, since that doesn't imply that it's _not_ being created here the way "register" does.

And so maybe "add" will work for the derived source too, then, actually... it's kind of neutral about the whole point of when it's being created.